client/src/pages/Profile/EditProfile/EditProfile.tsx

```diff
@@ -0,0 +1,32 @@
+import Grid from '@material-ui/core/Grid';
+import Sidebar from '../../../components/ProfileSidebar/Sidebar';
+import Box from '@material-ui/core/Box';
+import useStyles from '../useStyles';
+import Typography from '@material-ui/core/Typography';
+
+const EditProfile = (): JSX.Element => {
+  const classes = useStyles();
+  return (
+    <Grid container className={classes.root}>
+      <Grid xs={12} sm={3} item className={classes.sidebar}>
+        <Sidebar />
```

Comment:
Suggestion: rather than rendering the sidebar in every component, why don't we have a parent component which renders the sidebar and based on the link clicked it renders the relevant component (hint: you can use a switch case). This will make this feature easily scalable, what do you think? 😄

Reply:
Yes, This is a good idea.